automatic/selenium.powershell/update.ps1

```diff
@@ -0,0 +1,74 @@
+﻿param(
+    [string]$ModuleName = 'Selenium'
+)
+Import-Module Chocolatey-AU
+
+function global:au_SearchReplace {
+  @{
+    ".\legal\VERIFICATION.txt"      = @{
+      # Replace the module version on the Save-Module line
+      "(?i)(^\s*\-\s*Save\-Module.*\-Version\s+).*" = "`${1}$($Latest.ModuleVersion)"
+
+      # Attempt to remove all existing checksums
+      "(?i)(^\s*file\s*name\:).*"         = ""
+      "(?i)(^\s*checksum\s*type\:).*"     = ""
+      "(?i)(^\s*checksum(32)?\:).*"       = ""
+
+      # We will clean up the blank lines in the afterupdate step
+      # This is because this method doesn't support multiline matching
+
+      # Finally, inject the fresh checksums
+      "(?i)(^\s*3. The checksums should match the following:)" = "`${1}`n`n$(
+        @($Latest.ModuleChecksums.ForEach{
+          @(
+            "  file name: $($_.Path -replace "^(?<BasePath>.+)\\Selenium\\$($Latest.ModuleVersion)\\")"
+            "  checksum type: $($_.Algorithm)"
+            "  checksum: $($_.Hash)"
+            "`n"
+          ) -join "`n"
+        }) -join ''
+      )"
```

Comment:
This would be better to be handled in a `global:au_AfterUpdate` instead of inline here.

The `au_SearchReplace` cmdlet should be as simple as possible, not include nested logic like this.

Reply:
I've moved this logic into the AfterUpdate.